src/IIS/Settings/Bindings/MsmqFormatNameBindingSettings.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Cake.IIS.Settings.Bindings.FluentAPI;
+
+namespace Cake.IIS.Settings.Bindings
+{
+    /// <summary>
+    /// Class to represent and configure msmq.formatname binding.
+    /// </summary>
+    public sealed class MsmqFormatNameBindingSettings : IMsmqFormatNameBindingSettings, IBindingSettings
```

Comment:
Why don't the individual bindings inherit from a base class since they share the majority of their properties?